source/_integrations/openplantbook.markdown

```diff
@@ -0,0 +1,117 @@
+---
+title: OpenPlantbook
+description: Instructions on how to setup OpenPlantbook with Home Assistant.
+ha_category:
+  - Environment
+ha_iot_class: Cloud Polling
+ha_release: 0.116
+ha_quality_scale: internal
```

Comment:
internal isn't the right option in my opinion, see: https://www.home-assistant.io/docs/quality_scale/

Reply:
Sorry.  Missed that when copying from another integration